src/DipoleAlgorithms.cc

```diff
@@ -491,34 +363,30 @@ std::pair<double,int> PsfDipoleFlux::chi2(
     return std::pair<double,int>(chi2, nPix);
 }    
  
-template<typename PixelT>
-void PsfDipoleFlux::_apply(
-    afw::table::SourceRecord & source, 
-    afw::image::Exposure<PixelT> const& exposure,
-    afw::geom::Point2D const & center // Not used; source required to have footprint&peaks
+void PsfDipoleFlux::measure(
```

Comment:
Here, at least, it's because we've changed the interface in the base class to just take a single-precision float.  That's all we were using in practice, and there was quite a bit of code complexity involved in simulating a templated virtual method, which we've since been able to simplify.